avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -112,7 +112,8 @@ Describe 'File/folder tests' -Tag 'Modules' {
       # =========
       try {
         $rawData = Invoke-WebRequest -Uri $telemetryCsvLink
-      } catch {
+      }
```

Comment:
Just earlier today, we merged a PR enforcing PowerShell formatting that should avoid these changes. Could you please merge latest back to you fork branch, add a space anywhere to this file and save? That should enforce the new aligned formatting to this file. Thanks!